python/core/classes-ii/method-objects.md

```diff
@@ -86,17 +86,17 @@ Store a method object of class `Hello` in the variable `temp` and call it afterw
 
 ```python
 class Hello:
-  def greet(self):
-    print('Hello everyone')
+  def greet(name):
+      print(f'Hello {name}!')
```

Comment:
do we ever explain how printf works in python? we should verify that this concept is explained beforehand, so that it won't create any confusion for users

Reply:
@Stefan-Stojanovic can you check the Python content (string formatting insights are a good start) for this concept?

Reply:
I can't find anything

Reply:
then I don't think it is a good idea to include it as question. can you re-write the question so that it doesn't use f strings? @Stefan-Stojanovic

Reply:
I've looked into this, I don't think we should remove the print function. There were only ever 2 comments that got confused with it. Maybe I can add a footnote either in this insight, or the insight this was first used on?